src/main/java/com/oneops/api/resource/Assembly.java

```diff
@@ -312,4 +285,136 @@ public CiResource deleteAssembly(String assemblyName) throws OneOpsClientAPIExce
 		String msg = String.format("Failed to delete assembly with name %s due to null response", assemblyName);
 		throw new OneOpsClientAPIException(msg);
 	}
+
+
+	/**
+	 * AddTeams for the given assembly
+	 *
+	 * @param orgName      {mandatory}
+	 * @param assemblyName {mandatory}
+	 * @param teams        {mandatory}
+	 * @return teamList
+	 * @throws OneOpsClientAPIException
+	 */
+	public List<CiResource> addTeamsByAssembly(String orgName, String assemblyName, List<String> teams) throws OneOpsClientAPIException {
```

Comment:
Both methods `addTeamsByAssembly` and `addingTeamNameByAssembly` have the same signature and return type. Please merge these into a single method by name `addTeamsToAssembly`

Reply:
Pushed the PR review changes and the PR is https://github.com/oneops/OneOpsAPIClient/pull/51